partiql-eval/src/test/kotlin/org/partiql/eval/internal/exclude/CompiledExcludeItemTest.kt

```diff
@@ -0,0 +1,251 @@
+package org.partiql.eval.internal.exclude
```

Comment:
These tests are the same as the commented out tests from `partiql-eval/src/test/kotlin/org/partiql/lang/eval/EvaluatingCompilerExcludeTests.kt`